src/data/de/index.ts

```diff
@@ -314,21 +314,21 @@ export const instanceData = {
     events: {
       entityInParentPreposition: 'in',
       commentInParentPreposition: "zu",
-      setThreadStateArchived: "%actor% hat eine %thread% archiviert.",
-      setThreadStateUnarchived: "%actor% hat eine %thread% aus dem Archiv geholt.",
-      createComment: "%actor% hat einen %comment% in einer %thread% erstellt.",
-      createThread: "%actor% hat eine %thread% in %object% erstellt.",
+      setThreadStateArchived: "%actor% hat %thread% archiviert.",
```

Comment:
Only change the `en/index.ts` all other files are generated by [crowdin](https://crowdin.com/translate/serlo/76/en-de). Sorry to say but this will be overwritten and have to be done again in the tool.

Reply:
I added a warning in https://github.com/serlo/frontend/wiki/Internationalization.

Reply:
Oh, seems like I missed that. I rolled back the changes to de/index.ts. 

I can also do the changes in Crowdin but I don't know if I have access.

Reply:
I think you can use your GitHub account to sign in?
If not ping me and I figure out the access :)

Reply:
(but this happens after we merge it to staging)